browser/src/DatasetSelector.tsx

```diff
@@ -627,38 +639,35 @@ const DatasetSelector = withRouter(({ datasetOptions, history, selectedDataset }
             description: `${sampleCounts.gnomad_sv_r2_1_controls.total.toLocaleString()} samples`,
             childReferenceGenome: referenceGenome('gnomad_sv_r2_1_controls'),
           },
-        ],
-      }
-    )
-  }
-
-  if (includeCopyNumberVariants) {
-    const topLevelCopyNumberVariantDataset = hasCopyNumberVariants(selectedDataset)
-      ? selectedDataset
-      : 'gnomad_cnv_r4'
-
-    datasets.push(
-      {
-        id: 'current_cnv_dataset',
-        isActive: hasCopyNumberVariants(selectedDataset),
-        label: labelForDataset(topLevelCopyNumberVariantDataset),
-        url: datasetLink(topLevelCopyNumberVariantDataset),
-      },
-      {
-        id: 'other_cnv_variant',
-        isActive: hasCopyNumberVariants(selectedDataset),
-        label: 'More datasets',
-        children: [
           {
             id: 'gnomad_cnv_r4',
             label: labelForDataset('gnomad_cnv_r4'),
             url: datasetLink('gnomad_cnv_r4'),
-            description: `samples`,
-            // description: `${sampleCounts.gnomad_cnv_r4.total.toLocaleString()} samples`,
+            description: `${sampleCounts.gnomad_cnv_r4.total.toLocaleString()} samples`,
             childReferenceGenome: referenceGenome('gnomad_cnv_r4'),
           },
         ],
-      }
+      },
```

Comment:
~~this commented-out chunk can come out~~ and so it does, in a later commit. This spot is a good example of the value of tidying your commit history via rebase, since I spent a little time here reviewing a piece of code that later was cut, meaning that time was wasted.